core/primitives/src/views.rs

```diff
@@ -2624,7 +2624,8 @@ impl From<ExtCostsConfigView> for near_primitives_core::config::ExtCostsConfig {
                 ExtCosts::alt_bn128_g1_sum_element => view.alt_bn128_g1_sum_element,
                 ExtCosts::alt_bn128_pairing_check_base => view.alt_bn128_pairing_check_base,
                 ExtCosts::alt_bn128_pairing_check_element => view.alt_bn128_pairing_check_element,
-        };
+        }
+        .map(|_, value| WeightedGas { base: value, weight: 1.into() });
```

Comment:
This might be fine for now but it makes the conversion from config to view back to config lossy. We should check if we have some tests that would fail because of that. If so, we need to figure out how to deal with that test. (I don't want tests to randomly start failing because we set the first weight != 1)

Reply:
I'll investigate such failures now, but I think an important question here is why do we even have a conversion back from the ConfigView to the Config? Is it used anywhere in a meaningful way? I'll try to disable it and see what breaks.

Reply:
1. So far, everything in the configuration was interesting enough to users that we wanted it to be reflected in the view and it's easy to test that by converting it back. (This argument _might_ no longer holds with the addition of param weights).
2. For some reasons I am not familiar, the Rosetta RPC adapter constructs a `RuntimeConfig` from an RPC call. There we actually need this conversion in production code.(This seems worth fixing but I don't know how much effort it would be.)

Reply:
I've added non-default compute cost to one of the parameters and verified that no tests in nearcore fail. I've also reviewed the code usage of `RuntimeConfigView`[1] and haven't found any examples where we do a conversion back from the view into config. I'll try removing the `From` in that direction and see if anything breaks and also will ask on Zulip in case anyone knows why it exists.

[1] https://cs.github.com/?scope=org%3Anear&scopeName=near&q=%22RuntimeConfigView%22

Reply:
Hm, maybe the usage in rosetta has been removed. Even better :D

If you can remove it and all tests pass, go for it. I don't think we have to check further if it works in the latest code base. Otherwise, if for some reason we really need the conversion, I'd vote to remove the `From` impl anyway and replace it with a constructor that clearly indicates in the name that it is a lossy conversion.